### PR TITLE
update mvn dependencies

### PR DESCRIPTION
**Pull request by ahorek**

### CODE REVIEW

File: core/pom.rb [hunk 1]
```diff
@@ -172,7 +172,7 @@
   plugin( :compiler,
           'encoding' => 'utf-8',
           'debug' => 'true',
-          'verbose' => 'true',
+          'verbose' => 'false',
           'fork' => 'true',
```
Comment (headius) [inline core/pom.rb:176]: What is the purpose of enabling verbose output here?

Comment (ahorek) [reply core/pom.rb:176]: plugin :compiler in the latest version prints too verbose output, this restores the previous behavior

Comment (headius) [reply core/pom.rb:176]: Oh ok I see. They made verbose more verbose and we don't need it.

Comment (ahorek) [reply core/pom.rb:176]: just noisy loading messages, nothing relevant
```
[INFO] [loading C:\jruby\core\target\classes\org\jruby\Ruby.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\RubyModule.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\RubyClass.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\anno\TypePopulator.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\internal\runtime\methods\JavaMethod.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\internal\runtime\methods\DynamicMethod.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\runtime\Arity.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\runtime\Visibility.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\runtime\MethodIndex.class]
[INFO] [loading /modules/java.base/java/util/Arrays.class]
[INFO] [loading /modules/java.base/java/util/List.class]
[INFO] [loading c:\maven\builds\javax\annotation\javax.annotation-api\1.3.1\javax.annotation-api-1.3.1.jar(/javax/annotation/Generated.class)]
[INFO] [loading /modules/java.base/java/lang/Object.class]
[INFO] [loading /modules/java.base/java/lang/SuppressWarnings.class]
[INFO] [loading /modules/java.base/java/lang/Class.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\anno\JRubyClass.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\anno\JRubyMethod.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\anno\FrameField.class]
[INFO] [loading /modules/java.base/java/lang/annotation/Retention.class]
[INFO] [loading /modules/java.base/java/lang/annotation/RetentionPolicy.class]
[INFO] [loading /modules/java.base/java/lang/annotation/Target.class]
[INFO] [loading /modules/java.base/java/lang/annotation/ElementType.class]
[INFO] [loading /modules/java.base/java/lang/Deprecated.class]
[INFO] [loading /modules/java.base/java/lang/annotation/Annotation.class]
[INFO] [loading /modules/java.base/java/lang/String.class]
[INFO] [loading /modules/java.base/jdk/internal/PreviewFeature.class]
[INFO] [loading /modules/java.base/jdk/internal/PreviewFeature$Feature.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\CompatVersion.class]
[INFO] [checking org.jruby.gen.org$jruby$RubyArray$POPULATOR]
[INFO] [loading /modules/java.base/java/io/Serializable.class]
[INFO] [loading /modules/java.base/java/lang/AutoCloseable.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\RubyObject.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\RubyBasicObject.class]
[INFO] [loading /modules/java.base/java/lang/Cloneable.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\runtime\builtin\IRubyObject.class]
[INFO] [loading /modules/java.base/java/lang/Comparable.class]
[INFO] [loading C:\jruby\core\target\classes\org\jruby\runtime\marshal\CoreObjectType.class]
...
...
```
...

**Pull request merged**